gobblin-cluster/src/main/java/org/apache/gobblin/cluster/HelixUtils.java

```diff
@@ -278,6 +278,7 @@ static void waitJobCompletion(HelixManager helixManager, String workFlowName, St
           case STOPPING:
             log.info("Waiting for job {} to complete... State - {}", jobName, jobState);
             Thread.sleep(TimeUnit.SECONDS.toMillis(1L));
+            // TODO: fix the incorrect stoppingStateEndTime, and revisit GOBBLIN-1692
```

Comment:
Ideally we should have a gobblin ticket for this TODO to better outline the context of why we didn't do it now and when we should do it in the future